superdesk/users/users.py

```diff
@@ -15,112 +15,113 @@
 
 
 class UsersResource(Resource):
+    readonly = True
 
-    def __init__(self, endpoint_name, app, service, endpoint_schema=None):
-        self.readonly = True if app.config.get('LDAP_SERVER', None) else False
```

Comment:
@mdhaman Apparently the `__init__` method was not the problem and I managed to solve the issue by modifying the liveblog files only (fortunately). I'll close this PR since it is not useful anymore.